_posts/2018-01-05-python101.md

```diff
@@ -0,0 +1,734 @@
+---
+layout: page
+title: Programming Crash Course: Python 101
+categories:
+  - mac
+  - windows
+  - gnu-linux
+hierarchy:
+  -
+    name: "Topics"
+    path: "/topics"
+---
+
+Hey there!
+
+We’re going to crash course through the programming language **Python** to introduce you to the very basics of coding. If you have no or minimal coding background, then this is the place for you! Otherwise, feel free to move on and explore the rest of StartHacking.
+
+# Table of Contents
+1. Basics
+2. Types
+3. Expressions
+4. Relationships/Booleans
+5. Variables
+6. Sequences
+7. Strings
+8. Lists
+9. Tuples
+10. Conditionals
+11. If statements
+12. Boolean operations
+13. Looping
+14. For statements
+15. While statements
+16. Functions
+17. Classes
```

Comment:
Can we make the whole table of contents links to the appropriate sections? [Contributing](https://github.com/starthacking/starthacking.github.io/blob/master/CONTRIBUTING.md) has a lot of examples of linking to different sections in one page.